src/node_http_parser_impl.h

```diff
@@ -406,6 +409,10 @@ class Parser : public AsyncWrap, public StreamListener {
     return 0;
   }
 
+  AsyncWrap::ProviderType set_provider_type(AsyncWrap::ProviderType provider) {
+    return AsyncWrap::set_provider_type(provider);
+  }
```

Comment:
Why is this method necessary? Can’t we use the one we inherit from `AsyncWrap` directly?

Reply:
@mcollina You marked this as resolved, but it doesn’t seem to be resolved? These lines should just be removed.